bin/ChakraCore/CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ if(CMAKE_SYSTEM_NAME STREQUAL Linux)
   set(LINKER_END_GROUP
     -Wl,--no-whole-archive
     -Wl,--end-group
+    -static-libstdc++
```

Comment:
Apple doesn't support system libraries are statically linked and we target OSX 10.8 SDK.. So it is expected to work 10.8+